engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Error.java

```diff
@@ -25,6 +25,7 @@ public class Error {
   private final UninitializedState uninitializedState;
   private final NoSuchMethod noSuchMethod;
   private final NoSuchConversion noSuchConversion;
+  private final NoConversionCurrying noConversionCurrying;
```

Comment:
There already is `NoConversionCurrying` builtin!? That'd be surprising.

I don't think we should introduce new builtin (the less builtins, the better). Especially if this is not a permanent state (the description indicates follow up work is going to be needed).

Just reuse `NoSuchConversion` with different message, in my opinion.

Reply:
I forgot to add it to git, fixed.

I'm not entirely convinced about merging it into one. No conversion vs no conversion currying are two different things really.